plugins/faustwp/includes/settings/callbacks.php

```diff
@@ -11,6 +11,26 @@
 	exit;
 }
 
+add_action( 'admin_notices', __NAMESPACE__ . '\\frontend_url_notice' );
+/**
+ * Callback for WordPress 'admin_notices' action.
+ *
+ * Show frontend_url missing error if it doesn't exist.
+ *
+ * @return void
+ */
+function frontend_url_notice() {
+	$frontend_url_setting = faustwp_get_setting( 'frontend_uri' );
```

Comment:
We should use a guard clause here to ensure the notice will only shown on pages owned by the faustwp plugin.

```suggestion
function frontend_url_notice() {
	$screen = get_current_screen();

	// Exit if not this plugin's settings page.
	if ( $screen->id !== 'settings_page_faustwp-settings' ) {
		return;
	}

	$frontend_url_setting = faustwp_get_setting( 'frontend_uri' );
```